app/src/main/java/com/dicoding/picodiploma/sejiwaproject/MainActivity.kt

```diff
@@ -2,11 +2,46 @@ package com.dicoding.picodiploma.sejiwaproject
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.dicoding.picodiploma.sejiwaproject.adapter.LeagueAdapter
+import com.dicoding.picodiploma.sejiwaproject.db.LeagueData
+import com.dicoding.picodiploma.sejiwaproject.model.League
+import org.jetbrains.anko.*
+import org.jetbrains.anko.recyclerview.v7.recyclerView
 
 class MainActivity : AppCompatActivity() {
+    private lateinit var rvLeague: RecyclerView
+    private var list: ArrayList<League> = arrayListOf()
+
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        MainActivityUI().setContentView(this)
+
+        rvLeague = findViewById(R.id.rv_league)
+        rvLeague.setHasFixedSize(true)
+
+        list.addAll(LeagueData.listData)
+        showRecyclerList()
+    }
+
+    class MainActivityUI : AnkoComponent<MainActivity> {
```

Comment:
You have separate `ItemLeagueUI` and `DetailLeagueUI` to another file. Why you don't do it for this view?

Reply:
MainActivityUI class has been separated with MainActivity class